packages/components/pie-link/test/component/pie-link.spec.ts

```diff
@@ -1,58 +1,61 @@
-import { test, expect } from '@sand4rt/experimental-ct-web';
-import { PieLink, type LinkProps } from '../../src/index.ts';
-import { tags } from '../../src/defs.ts';
+import { test, expect } from '@playwright/test';
+import { BasePage } from '@justeattakeaway/pie-webc-testing/src/helpers/page-object/base-page.ts';
+import { type LinkProps, tags } from '../../src/defs.ts';
 
-const componentSelector = '[data-test-id="pie-link"]';
+import { link } from '../helpers/page-object/selectors.ts';
 
-const anchorProps: LinkProps = {
-    tag: 'a',
-    href: '#',
-    target: '_blank',
-    rel: 'nofollow',
-};
+const componentSelector = '[data-test-id="pie-link"]';
 
 test.describe('PieLink - Component tests', () => {
-    test('should be visible', async ({ mount, page }) => {
+    test('should be visible', async ({ page }) => {
         // Arrange
-        await mount(PieLink, {
-            slots: { default: 'Link!' },
-        });
+        const linkPage = new BasePage(page, 'link');
+        const props: LinkProps & { slot: string } = {
+            slot: 'Link',
+        };
+
+        await linkPage.load({ ...props });
 
         // Act
-        const link = page.locator(componentSelector);
+        const linkComponent = page.getByTestId(link.selectors.container.dataTestId);
 
         // Assert
-        await expect(link).toBeVisible();
+        await expect(linkComponent).toBeVisible();
     });
 
-    test('should render as anchor when tag="a"', async ({ mount, page }) => {
+    test('should render as anchor when tag="a"', async ({ page }) => {
         // Arrange
-        await mount(PieLink, {
-            props: {
-                ...anchorProps,
-            },
-            slots: { default: 'Anchor Link!' },
-        });
+        const linkPage = new BasePage(page, 'link');
+        const props: LinkProps & { slot: string } = {
+            tag: 'a',
+            href: 'https://pie.design',
+            target: '_blank',
+            rel: 'nofollow',
+            slot: 'Anchor Link',
+        };
+
+        await linkPage.load({ ...props });
 
         // Act
         const link = page.locator(`a${componentSelector}`);
```

Comment:
May be a better approach to remove the hardcoded selector.
```suggestion
        const link = page.getByTestId(`link.selectors.container.dataTestId`).locator('a');
```